### PR TITLE
improvement(kms): add one more variation of the KMS nemesis

### PR DESCRIPTION
**Pull request by vponomaryov**
Before we had only one variation - without KMS key rotation.
So, add one more - with KMS key rotation.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [ ] ~~I added the relevant `backport` labels~~
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: data_dir/nemesis.yml [hunk 1]
```diff
@@ -112,7 +112,7 @@
   - limited = True
 - disrupt_mgmt_restore:
   - disruptive = True
-  - kubernetes = False
```
Comment (fruch) [inline data_dir/nemesis.yml:115]: why are touching this one ?

**Comment by fruch** [comment]
@vponomaryov rerun the unittest that create the nemesis list, the new nemesis are missing from it

File: data_dir/nemesis.yml (hunk 1 above)
Comment (fruch) [reply data_dir/nemesis.yml:115]: I'm guessing someone update it,and didn't run the unittest again....

**Comment by vponomaryov** [comment]
> @vponomaryov rerun the unittest that create the nemesis list, the new nemesis are missing from it

The nemesis list you are referring to considers only those classes where we call disruption methods directly.
So, this change is correct according to the existing logic.

File: data_dir/nemesis.yml (hunk 1 above)
Comment (vponomaryov) [reply data_dir/nemesis.yml:115]: Yes, this diff is in master code.

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**